src/screens/send/components/fee-options/fee-options.tsx

```diff
@@ -60,33 +58,42 @@ export class FeeOptionsComponent extends React.Component<
         const blockchainInstance = getBlockchain(this.props.account.blockchain);
```

Comment:
please don't use arrow function in classes (please read: https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1)

Reply:
also try to use a verb in functions/methods names, in this case `getEstimatedFees()`